hasura.planx.uk/metadata/tables.yaml

```diff
@@ -252,6 +252,24 @@
       using:
         foreign_key_constraint_on: team_id
   insert_permissions:
+    - role: demoUser
```

Comment:
I know this is still a draft, but just a heads up that this is already an action granted by the `public` role and doesn't need to be repeated for the `demo` role.